<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgotPassword</class>
 <widget class="QDialog" name="forgotPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reset Passord</string>
  </property>
  <widget class="QGroupBox" name="Footer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>341</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#3a444c;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget1_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>251</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="forgotPasswordCancelButton">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forgotPasswordResetButton">
       <property name="styleSheet">
        <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons White/Icons/White/Passsword.png</normaloff>:/icons White/Icons/White/Passsword.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/images/background.jpg);
background-position:center;
background-repeat: no-repeat;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:500;
color:#ffffff;
background: none;</string>
    </property>
    <property name="text">
     <string>Reset Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="accountTypegroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QLineEdit" name="lecturerForgotFormLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;
color: #333;</string>
    </property>
    <property name="placeholderText">
     <string>Lecturer Username</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="studentAccountTypeRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>74</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="lecturerAccountTypeRadioButton">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>50</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lecturer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentIdForgotFormLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;
color: #333;</string>
    </property>
    <property name="placeholderText">
     <string>Student ID #</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="securityQuestiongroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Security Questions</string>
   </property>
   <widget class="QLineEdit" name="QuestionAnswer_forgotFormLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;
color: #333;</string>
    </property>
    <property name="placeholderText">
     <string>(Enter your Answer)</string>
    </property>
   </widget>
   <widget class="QLabel" name="Question_ForgotForm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="text">
     <string>Question:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="securityQuestionsForgotFormcomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#ffffff;
color:#333;</string>
    </property>
    <item>
     <property name="text">
      <string>Select Question</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="checkAnswerResetButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Check My Answer</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/icons White/Icons/White/Single Arrow RIGHT.png</normaloff>:/icons White/Icons/White/Single Arrow RIGHT.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="resetPasswordGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Reset Password</string>
   </property>
   <widget class="QLineEdit" name="newPassword_forgotFormLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;
color: #333;</string>
    </property>
    <property name="placeholderText">
     <string>(Enter new password)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPassword_forgotFormLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;
color: #333;</string>
    </property>
    <property name="placeholderText">
     <string>( Confirm password)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="accountLostResetButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Search Account</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons White/Icons/White/Search.png</normaloff>:/icons White/Icons/White/Search.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
